config/socket/game.js

```diff
@@ -260,18 +260,19 @@ Game.prototype.shuffleCards = function(cards) {
 };
 
 Game.prototype.dealAnswers = function(maxAnswers) {
+  var self = this;
   maxAnswers = maxAnswers || 10;
   var storeAnswers = function(err, data) {
-    this.answers = data;
+    self.answers = data;
   };
-  for (var i = 0; i < this.players.length; i++) {
-    while (this.players[i].hand.length < maxAnswers) {
-      this.players[i].hand.push(this.answers.pop());
+  this.players.map((player) => {
```

Comment:
Great job refactoring this to use map as opposed to a regular iterator

Reply:
Thanks for the feedback